stwcs/updatewcs/astrometry_utils.py

```diff
@@ -580,44 +581,116 @@ def apply_new_apriori(self, obsname):
 
         return wname
 
-    def isAvailable(self):
-        """Test availability of astrometryDB web-service."""
+    def isAvailable(self, max_tries=2, testing=False):
```

Comment:
This adds new parameters but they are not set from any calling code. Is the intention to always use the default parameters to be used in OPS?